src/loaders.py

```diff
@@ -32,23 +35,26 @@ def __init__(self, ids, annotations, annotations_human_labels, images_dir, targe
         self.target_encoder = target_encoder
         self.train_mode = train_mode
 
+        self.image_transform = image_transform
+
     def __len__(self):
         return len(self.ids)
 
     def __getitem__(self, index):
-
         Xi = self.load_from_disk(index)
 
+        Xi = self.image_transform(Xi)
```

Comment:
@taraspiotr I think `image_transform` along `image_transform_with_target` should be used here to make the transformations are performed both on image and keypoints (soon)

Reply:
@jakubczakon yes, that's true, and the same goes for augmentations, but for now we only do a resize which doesn't affect keypoints with relative coordinates